packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -275,18 +274,7 @@ export class ObjectSetListenerWebsocket {
     // so we filter those out.
     const readySubs = [...this.#subscriptions.values()].filter(isReady);
 
-    if (readySubs.length === 0) {
```

Comment:
This is not the behavior we want. We want to update that we have no more subscriptions so that we don't unnecessarily take up space over the wire